datasets/oneoff/usc-049.id295a8bc5-2a73-11ef-ba6d-a9961dfb011e_0_0.cgel

```diff
@@ -0,0 +1,98 @@
+        :Head (VP
+            :Head (VP
+                :Head (V_aux :t "is" :xpos "VBZ" :l "be")
+                :PredComp (Clause
+                    :Head (VP
+                        :Head (V :t "authorized" :xpos "VBN" :l "authorize")
+                        :Comp (Clause
+                            :Head (VP
+                                :Marker (Sdr :t "to")
+                                :Head (VP
+                                    :Head (V_aux :t "be" :xpos "VB")
+                                    :Comp (Clause
+                                        :Head (VP
+                                            :Head (V :t "appropriated" :p "," :xpos "VBN" :l "appropriate")
+                                            :Supplement (PP
+                                                :Head (P :t "out")
+                                                :Comp (PP
+                                                    :Head (P :t "of")
+                                                    :Obj (NP
+                                                        :Det (DP
+                                                            :Head (D :t "any"))
+                                                        :Head (Nom
+                                                            :Head (Nom
+                                                                :Head (N :t "money")
+                                                                :Mod (PP
+                                                                    :Head (P :t "in")
+                                                                    :Obj (NP
+                                                                        :Det (DP
+                                                                            :Head (D :t "the"))
+                                                                        :Head (Nom
+                                                                            :Head (N :t "Treasury")))))
+                                                            :Mod (Clause
+                                                                :Head (VP
+                                                                    :Mod (AdvP
+                                                                        :Head (Adv :t "not"))
+                                                                    :Head (VP
+                                                                        :Mod (AdvP
+                                                                            :Head (Adv :t "otherwise"))
+                                                                        :Head (V :t "appropriated" :p "," :xpos "VBN" :l "appropriate")))))))))))))))
+                :DisplacedSubj (NP
+                    :Head (Nom
+                        :Mod (AdjP
+                            :Head (Adj :t "such"))
+                        :Head (N :t "amounts" :l "amount"))))
+            :Mod (PP
```

Comment:
> Note: cgel validator doesn't like the PP indirect complement:

Maybe treat the from-PP as a Supplement? https://github.com/nert-nlp/cgel/issues/143